testdata/p4_16_samples_outputs/extract_struct-midend.p4

```diff
@@ -0,0 +1,41 @@
+#include <core.p4>
+
+struct Tst {
+    bit<32> sdata;
+}
+
+header Header {
+    bit<32> data;
+    Tst     t;
+}
+
+parser p0(packet_in p, out Header h) {
+    bool b_0;
+    state start {
+        b_0 = true;
+        p.extract<Header>(h);
+        p.extract<Tst>(h.t);
```

Comment:
This looks wrong. BMv2 will not support extraction to structs.
If you want to allow extraction to arbitrary structs you should create a pass like midend/expandEmit, but which handles extracts. 

You should also add a test case for bmv2 which extracts to structs.

Reply:
Sorry, this set of files need to be removed.  Very early on, during this work, the PR was changing extract behavior, but then the feedback provided ask to not change the behavior.